config/module.config.php

```diff
@@ -25,6 +25,7 @@
                 'SlmQueue\Strategy\MaxRunsStrategy' => array('max_runs' => 100000),
                 'SlmQueue\Strategy\MaxMemoryStrategy' => array('max_memory' => 100 * 1024 * 1024),
                 'SlmQueue\Strategy\InterruptStrategy',
+                'SlmQueue\Strategy\MaxPollingFrequencyStrategy' => array('max_frequency' => 0)
```

Comment:
Not sure if it should be added it to all workers per default. Consider a use case where a SQS and a Doctrine Queue are defined. For the doctrine adapter it makes no sense to sleep on the PROCESS_QUEUE event as that would considerable slow down the queue if there are jobs in it. 
I would provide it as a tool for usage just as the FileWatchStrategy is. It's there to be used when the developer needs it, but not enabled by default.